Dockerfile

```diff
@@ -6,9 +6,15 @@ RUN CGO_ENABLED=0 go build
 
 FROM alpine:3.11
 EXPOSE 8083
-RUN apk add git
+RUN apk add git openssh python3 py3-pip
+RUN pip3 install --upgrade pyyaml
+RUN mkdir -p /root/.ssh/ && \
+    ssh-keyscan github.com >> /root/.ssh/known_hosts && \
```

Comment:
Good stuff, but, I'm just thinking, might it be that github changes IPs from time to time. If it's so then we'll need to rebuild an image when this happens. But if we add this into entrypoint then only container restart will require to handle such case.

Reply:
Good point. I'll move it to a new entrypoint script.

Reply:
Done.